cmd/skopeo/proxy.go

```diff
@@ -182,6 +204,30 @@ type convertedLayerInfo struct {
 	MediaType string        `json:"media_type"`
 }
 
+// mapProxyErrorCode turns an error into a known string value.
+func mapProxyErrorCode(err error) string {
+	if err == nil {
```

Comment:
(Absolutely non-blocking: Using a Go `switch` would be a tiny bit shorter. This works just fine.)